lms/product/plugin/misc.py

```diff
@@ -0,0 +1,26 @@
+class MiscPlugin:
+    """
+    Plugin for product differences that don't have a better abstractions around them.
+
+    Works as a placeholder location to centralize LMS differences here without the need to create
+    bigger abstractions before it's clear what that abstraction should look like.
+
+    The aim also includes to remove any checks around product.family around the code base.
+
+    New methods here should not try to get a very tight API as easier to refactor once multiple
+    LMS have the same issue vs getting the right parameters in the first occurrence.
+
+    Once any of these is implemented by more than one product or a group of
+    methods looks like it could belong to their own plugin it's time to refactor them out.
+    """
+
+    def post_configure_assignment(self, request):  # pragma: nocover
+        """Run any actions needed after configuring an assignment."""
```

Comment:
Worth putting any advice about what _not_ to do in here? We have policies about not making loads of synchronous calls in general.

Or should we just trust our own good judgement?

Reply:
Not sure, this is mostly about what you'll do for an specific product, I reckon the same judgement should apply here than in the rest of the code base.